### PR TITLE
Fix Issue 2718

### PR DESCRIPTION
**Pull request by MiloTodt**
Added "Check for Updates" to the Help menu, which manually calls the built in update function as requested.

Fix > #2718 

image

### CODE REVIEW

File: browser/components/MarkdownPreview.js [hunk 1]
```diff
@@ -491,6 +491,7 @@ export default class MarkdownPreview extends React.Component {
     eventEmitter.on('export:save-md', this.saveAsMdHandler)
     eventEmitter.on('export:save-html', this.saveAsHtmlHandler)
     eventEmitter.on('print', this.printHandler)
+    eventEmitter.on('check-updates', this.checkUpdate)
```
Comment (ZeroX-DG) [inline browser/components/MarkdownPreview.js]: I don't see the `checkUpdate` method in this file, is this a mistake? If it not can you explain this to me?

**Comment by MiloTodt** [comment]
I suppose it may help if I actually submit the code, eh?

File: browser/components/MarkdownPreview.js (hunk 1 above)
Comment (MiloTodt) [reply browser/components/MarkdownPreview.js]: Actually pushing the written code would have been a great first step on my end.

**Comment by MiloTodt** [comment]
##Diagram of event messages sent. 

The reason it goes menu->index->app->index is that I was unable to find a way to ssend from menu->app directly.

image

**Comment by ZeroX-DG** [comment]
I think it looks OK but I can't really test it because I'm on Linux. I'll need @Rokt33r to review this.

**Comment by Rokt33r** [comment]
I made a new spec, https://github.com/BoostIO/Boostnote/issues/2827, to resolve the issue. So I think I should close this pr. 😭 

Thanks for your contribution!

**Pull request closed without merging**